.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ jobs:
             composer config repositories.systemseed/anu_lms path ../anu_lms && \
             composer config minimum-stability dev && \
             composer require "systemseed/anu_lms:*@dev" && \
-            composer require --dev drupal/core-dev:^9 && \
+            composer require --dev drupal/core-dev:^9 drupal/pathauto && \
```

Comment:
How should we install on local for running tests? I tried `composer -- require systemseed/anu_lms:dev-js-tests --prefer-source` and it did NOT download pathauto. I had to manually install it.